tools/cli/internal/changelog/outputfilter/operationconfig.go

```diff
@@ -0,0 +1,120 @@
+package outputfilter
+
+import (
+	"github.com/getkin/kin-openapi/openapi3"
+	"github.com/tufin/oasdiff/load"
+)
+
+type OperationConfig struct {
+	Path                   string
+	HTTPMethod             string
+	Tag                    string
+	Sunset                 string
+	ManualChangelogEntries map[string]string
+}
+
+type OperationConfigs struct {
+	Base     *OperationConfig
+	Revision *OperationConfig
+}
+
+func (e *OperationConfigs) Tag() string {
+	if e.Revision != nil {
+		return e.Revision.Tag
+	}
+
+	if e.Base != nil {
+		return e.Base.Tag
+	}
+
+	return ""
+}
+
+func (e *OperationConfigs) Sunset() string {
+	if e.Revision != nil {
+		return e.Revision.Sunset
+	}
+
+	if e.Base != nil {
+		return e.Base.Sunset
+	}
+
+	return ""
+}
+
+// newEndpointsConfigGivenBaseAndRevision parses the base and revision openapi specs
+// and returns the mapping between API operationId and EndpointConfig.
+func newEndpointsConfigGivenBaseAndRevision(base, revision *load.SpecInfo) map[string]*OperationConfigs {
```

Comment:
[nit] you're calling this newEndpointConfig then OperationConfigs in the struct, can we pick one?

Reply:
```suggestion
func newOperationConfigs(base, revision *load.SpecInfo) map[string]*OperationConfigs {
```

Reply:
I forgot to rename it 😿  on it

Reply:
updated it